### PR TITLE
Portfolio: accessibility

### PR DESCRIPTION
**Pull request by vabuyia**
-Fields that is required have been clearly been indicated, and asterisks * included in the field label.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,394 +2,275 @@
...
-        
-        <section class="last-content">
-          <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-        </section class="last-form">
-
-        <form action="https://formspree.io/f/xqkwojbj" class="form-contact" method="post">
-          <ul>
-            <li><input type="text" placeholder="Yeremias" name="name" id="name_form" maxlength="30" required/></li>
-            <li> <input type="email" placeholder="natajayanj14@gmail.com" name="email" id="email_form" required/> </li>
-            <li> <textarea name="message" placeholder="Write your message here"id="message" cols="30" rows="10" maxlength="500" required ></textarea></li>
```
Comment (VanessaAoki) [inline index.html:387]: - [ ] Every form fields and forms in the page should have basic validations, you can read more about it [here](https://www.w3.org/WAI/test-evaluate/preliminary/#forms).

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @vabuyia, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 
- [x] Your accessibility report: ✔️ Page title ✔️ Image text alt ✔️ Text headings ✔️ Color contrast ✔️ Resize ✔️ Moving content ✔️ Basic structure of the page 

## Required Changes ♻️
_Check the comments under the review._

so close

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (vabuyia) [reply index.html:387]: Field that are required have been clearly been indicated, and asterisks * included in the field label.

**Review by D4nielJ — APPROVED** [review]
## Hello @vabuyia, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

## Optional suggestions

I think you could also consider:
 - Restarting the README file with the correct use of the template provided [by Microverse here.](https://github.com/microverseinc/readme-template). Right now you're not using any of the features of .md files.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**